configure.ac

```diff
@@ -54,6 +54,31 @@ AC_ARG_WITH([modprobedir],
                             [Path to the system modprobe.d directory])],
             [modprobedir="$withval"], [modprobedir="/lib/modprobe.d"])
 AC_SUBST(modprobedir)
+AC_ARG_WITH([dbusdir],
+            [AS_HELP_STRING([--with-dbusdir],
+                            [Path to the system dbus-1 directory])],
+            [dbusdir="$withval"], [dbusdir="/usr/share/dbus-1"])
+AC_SUBST(dbusdir)
```

Comment:
Where is this used?

Reply:
It was used in Makefile.am, but I think it makes sense to just drop it.